inc/saiswitch.h

```diff
@@ -412,6 +412,12 @@ typedef enum _sai_switch_attr_t
        Default no map */
     SAI_SWITCH_ATTR_QOS_TC_AND_COLOR_TO_DSCP_MAP,
 
+    /** Number of preamble and Inter frame gap bytes to be added in the 
+     *  Meters(Policer, shaper and meter counters) [uint32_t]. 
+     *  Default to 20 Bytes. 
+     */
+    SAI_SWITCH_ATTR_QOS_METER_ADJUST,
```

Comment:
Based on MEF explanation, CIR includes the Preamble and and Inter frame gap in calculations, But some NPU's may/may not include this bytes in meter calculations by default. It will be controlled by external registers. Apps needs to have control to add/remove bytes from meter calculations.  
SAI model we can define instead of 20 bytes,  default behavior NPU should include these in  meter calculation. 
Preamble: 7 bytes Preamble
Start Frame Delimiter (SFD): 1 byte
Inter-frame Gap (IFG): 12 Byte(But it can be variable)